components/push/src/crypto.rs

```diff
@@ -188,7 +184,6 @@ impl Cryptography for Crypto {
         }
     }
 
-    // IIUC: objects created on one side of FFI can't be freed on the other side, so we have to use references (or clone)
```

Comment:
I wasn't able to map what this comment is saying to anything happening in the code below it, so I presumptuously deleted it.